inference-engine/ie_bridges/java/org/intel/openvino/TensorDesc.java

```diff
@@ -0,0 +1,68 @@
+package org.intel.openvino;
+
+import java.util.concurrent.BlockingDeque;
+
+public class TensorDesc extends IEWrapper {
+
+    public TensorDesc(long addr){
+       super(addr);
+    }
+
+    public TensorDesc(Precision precision, int[] dims, Layout layout) {
+        super(GetTensorDesc(precision.getValue(), dims, layout.getValue()));
+    }
+
+    public int[] getDims() {
+        return GetDims(nativeObj);
+    }
+
+    public void setDims(int[] dims) {
+        setDims(nativeObj, dims);
+    }
+
+    public void setLayout(Layout layout) {
+        setLayout(nativeObj, layout.getValue());
+    }
+
+    public Layout getLayout(){
+        return Layout.valueOf(getLayout(nativeObj));
+    }
+
+    public void setPrecision(Precision precision) {
+        setPrecision(nativeObj, precision.getValue());
+    }
+
+    public Precision getPrecision(){
+        return Precision.valueOf(getPrecision(nativeObj));
+    }
+
+    public void reshape(int[] dims, Layout layout){
+        reshape(nativeObj, dims, layout.getValue());
```

Comment:
@ilyachur @vinograd47 do we need to allow to change `TensorDesc` after creation? In java

Reply:
```reshape``` method is used in ```behcmark_app```  in ```Main.java```

Reply:
reshape for tensordesc or for CNNNetwork?

Reply:
No, this method isn't used